crossing-training.lic

```diff
@@ -518,12 +528,6 @@ class CrossingTraining
 
     success = cast?(data['cast'], data['symbiosis'], data['before'], data['after'])
 
-    wounds = check_health['wounds']
-    if !success && wounds.any? { |_k, v| v.include?('skin') }
-      wait_for_script_to_complete('safe-room', ['force'])
-      walk_to(@training_room)
```

Comment:
I noticed this `walk_to` isn't in the extracted method. Was that intentional?

Reply:
Dangit, no its not. Ill get that added back in.